ui/app/src/components/UnpublishedDashlet/UnpublishedDashlet.tsx

```diff
@@ -279,6 +294,7 @@ export function UnpublishedDashlet(props: UnpublishedDashletProps) {
                   </Typography>
                 }
               />
+              {over === item.path && <DashletItemOptions path={item.path} />}
```

Comment:
These should be the ListItem `secondaryAction`. Is that not available on `ListItemButton`? Also, the show hide might be achievable via css only so we can remove the hovering events.